app/src/lib/api.ts

```diff
@@ -112,13 +112,28 @@ export interface IAPIIssue {
 /** The combined state of a ref. */
 export type APIRefState = 'failure' | 'pending' | 'success'
 
+/**
+ * The API response for a combined view of a commit
+ * status for a given ref
+ */
+export interface IAPIRefStatusItem {
+  readonly created_at: string
```

Comment:
We've _generally_ leaned toward only including fields that we actually use in the API declarations.